website/docs/examples/multi_client_geoimage/README.md

```diff
@@ -0,0 +1,236 @@
+subscribe to the particular service. In the first place, we show how to filter the message type and
+if the message is of type `farm_ng.oak.proto.OakFrame` we store it in the queue.
+If the message is of type `farm_ng.gps.proto.GpsFrame` we try to find the closest image in the queue.
+
+In this example, we use the `get_stamp_by_semantics_and_clock_type` function to get
+the timestamp of the event. This function is used to get the timestamp of the event
+based on the semantics and the clock type. The semantics is the type of event and
+the clock type is the type of clock used to timestamp the event.
+
+The threshold used to synchronize the images and the gps data is defined by
+ the `stamp_diff` variable. This threshold depends on the precision of your application.
+
+For educational purposes, we left a lot of `print` statements in the code to
+visualize the flow of the program. We recommend to remove them later in your application.
+
+```python
+async def _subscribe(self, subscription: SubscribeRequest) -> None:
+  # the client name is the last part of the query
+  client_name: str = subscription.uri.query.split("=")[-1]
+  client: EventClient = self.clients[client_name]
+  # subscribe to the event
+  async for event, message in client.subscribe(subscription, decode=True):
+      print(f"Received event from {client_name}{event.uri.path}")
+      if isinstance(message, oak_pb2.OakFrame):
+          await self.image_queue.put((event, message))
+      elif isinstance(message, gps_pb2.GpsFrame):
+          stamp_gps = get_stamp_by_semantics_and_clock_type(
+              event, semantics="service/send", clock_type="monotonic"
+          )
+          if stamp_gps is None:
+              continue
+
+          geo_image = None
+
+          while self.image_queue.qsize() > 0:
+              event_image, image = await self.image_queue.get()
+              stamp_image = get_stamp_by_semantics_and_clock_type(
+                  event_image, semantics="service/send", clock_type="monotonic"
+              )
+              if stamp_image is None:
+                  continue
+              stamp_diff = abs(stamp_gps - stamp_image)
+
+              # NOTE: define this threshold depending on the precision of your application
+              if stamp_diff > 0.05:
+                  print(f"Skipping image because stamp_diff is too large: {stamp_diff}")
+                  continue
+              else:
+                  print(f"Synced image and gps data with stamp_diff: {stamp_diff}")
+                  geo_image = ((event_image, image), (event, message))
+                  break
+
+          if geo_image is None:
+              print("Could not sync image and gps data")
+              continue
+```
+
+In addition, we provide a `config.json` file example that contains the configuration of the
+service. Notice that the subscribers are against the `oak0` and `gps` services,
+in particular to the `/rgb` and `/pvt` paths respectively.
```

Comment:
I know it sounds obvious to us, but I would briefly mention why you're choosing `/pvt` instead of `/relposned`. Or at least leave a link for a tip.
Something like:

:::tip
If you want to better understand the different GPS message types, make sure to check this [link](https://amiga.farm-ng.com/docs/examples/file_reader_gps/).
:::